build-multiarch.sh

```diff
@@ -0,0 +1,63 @@
+TARGETIMAGE=${1:-target/image:ci}
+IS_RELEASE=${2:-no}
+WINBASE="mcr.microsoft.com/windows/nanoserver"
+OSVERSIONS=("1809" "1903" "1909" "ltsc2019" "2004" "20H2" "ltsc2022")
```

Comment:
Given the [compatibility matrix](https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2022%2Cwindows-11), suggested by @HofmeisterAn, shows there are two major Windows versions, I suggest keeping this Windows OS array smaller. Wdyt?
```suggestion
OSVERSIONS=("ltsc2019" "ltsc2022")
```

Reply:
It should be fine. The .NET SDK configuartion ships these versions: https://github.com/dotnet/dotnet-docker/tree/main/src/sdk/6.0. `1809` is `ltsc2019`?